manifests/foreman.pp

```diff
@@ -15,7 +15,7 @@
   $default_ca            = $certs::default_ca,
   $ca_key_password_file  = $certs::ca_key_password_file,
   $server_ca             = $certs::server_ca,
-) inherits certs::params {
+) inherits certs {
```

Comment:
It's not immediately obvious to me what this has to adding the foreman_spec.rb . Is this so that the test can `include certs::foreman` without having to specify params, and should this PR be labeled as 'enhancement' then?

Reply:
Without this, these tests will fail because it does not load `certs` class which there are parameters that it depends upon. So you could think of this as a bug fix.

Reply:
I also read this as a bugfix. It's mostly just chance we never ran into it before due to the way we called the code (if you first include certs, it does work).

Reply:
OK. Rather than creating a separate PR to get that to show up in changelogs, how about we create and link an issue?

This one can still be labeled to skip the changelog, and only the issue would show up.

Reply:
You can also make this just a bugfix PR and consider the acceptance tests as a regression test.

Reply:
Y'all just tell me what you want to see happen. For me this is about adding acceptance tests, and uncovered a chance bug fix that is required in order for this testing to work since it calls this class directly (which is not something we do in reality anywhere).

Reply:
I consider this a bugfix with a regression test, which to me is more important than only adding a regression test.

Reply:
I have split this single change out to it's own commit to fix the bug and then layered the new tests on top.